docs/src/examples.md

```diff
@@ -0,0 +1,160 @@
+# Examples
+
+* Create a blade.
+
+  ```julia
+  julia> vectors = Matrix([3 3; -4 -4; 0 1]);
+
+  julia> B = Blade(vectors)
+  Blade{Float64}(3, 2, [-0.6 0.0; 0.8 0.0; 0.0 -1.0], 5.0)
+
+  julia> dim(B)
+  3
+
+  julia> grade(B)
+  2
+
+  julia> norm(B)
+  5.0
+  ```
+
+* Compute the outer product between blades.
+
+  ```julia
+  julia> u = [3, -4, 0, 0];  # a vector is a 1-blade
+
+  julia> v = [3, -4, 1, 0];  # a vector is a 1-blade
+
+  julia> B = u ∧ v
+  Blade{Float64}(3, 2, [-0.6 0.0; 0.8 0.0; 0.0 -1.0], 5.0)
+
+  julia> grade(Blade(u))
+  1
+
+  julia> grade(Blade(v))
+  1
+
+  julia> grade(B)
+  2
+  ```
+
+  ```julia
+  julia> A = [0, 0, 0, 2];
+
+  julia> C = A ∧ B
+  Blade{Float64}(4, 3, [-0.6 0.0 0.0; 0.8 0.0 0.0; 0.0 1.0 0.0; 0.0 0.0 -1.0], -10.0)
+
+  julia> grade(Blade(A))
+  1
+
+  julia> grade(B)
+  2
+
+  julia> grade(C)
+  3
+  ```
+
+* Compute the additive and multiplicative inverses of a vector (1-blade).
+
+  ```julia
+  julia> v = Blade([3, 4, 0, 0])
+  Blade{Float64}(4, 1, [0.6; 0.8; 0.0; 0.0;;], 5.0)
+
+  julia> inverse(v)
+  Blade{Float64}(4, 1, [0.6; 0.8; 0.0; 0.0;;], -5.0)
+
+  julia> reciprocal(v)
```

Comment:
I renamed `reciprocal` to `inv` in PR #32 so if that PR looks good, I suggest replace `reciprocal` with `inv` here as well.